Makefile

```diff
@@ -342,23 +342,25 @@ benchmark:
 ###                                Linting                                  ###
 ###############################################################################
 
-golangci_lint_cmd=github.com/golangci/golangci-lint/cmd/golangci-lint
+golangci_lint_cmd=golangci-lint
```

Comment:
We could add the version number here too

Reply:
added